core/designsystem/src/main/res/values/strings.xml

```diff
@@ -16,4 +16,5 @@
     <string name="text_choose_file">Choose File</string>
     <string name="text_no_selected_file">No selected file…</string>
     <string name="text_unable_to_get_file_name">Unable to get file name</string>
+    <string name="plus_symbol_for_quantities">+</string>
```

Comment:
Here we need to add the new values that is going to be concatenated

Reply:
Done